srcpkgs/networkmanager-dmenu/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'networkmanager-dmenu'
+pkgname=networkmanager-dmenu
+version=2.1.0
+revision=1
+depends="python3 NetworkManager python3-gobject"
+short_desc="Control NetworkManager via dmenu"
+maintainer="Ishaan Bhimwal <ishaanbhimwal@protonmail.com>"
+license="MIT"
+homepage="https://github.com/firecat53/networkmanager-dmenu"
+distfiles="https://github.com/firecat53/networkmanager-dmenu/archive/refs/tags/v${version}.tar.gz"
+checksum=f39090bb924969dd5e76e8b07dd57ba89f146f444f225106383345d458de0729
+
+do_install() {
+	vbin networkmanager_dmenu
+	vlicense LICENSE.txt
+	vdoc README.md
+	vdoc config.ini.example
```

Comment:
this should probably be `vsconf` (see manual.md)

Reply:
it makes sense to put those in `/usr/share/examples` but unfortunately the original package doesn't do that and doing so will ultimately confuse the user about where to find what.

* [example of a similar void linux template](https://github.com/void-linux/void-packages/blob/master/srcpkgs/clipmenu/template)
* [original package aur pkgbuild](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=networkmanager-dmenu-git)

Reply:
it makes sense to put those in `/usr/share/examples` but unfortunately the original package doesn't do that and doing so will ultimately confuse the user about where to find what.

* similar void linux template [[1]](https://github.com/void-linux/void-packages/blob/master/srcpkgs/clipmenu/template)
* original package aur pkgbuild [[2]](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=networkmanager-dmenu-git)

Reply:
[1] doesn't install any example configuration at all
 
[2] is not void, arch may have different standards for package file locations (and that's AUR, which is more of a free-for-all than void-packages)

i still think that example configuration files should go in the place designated for example configuration files

Reply:
your idea seems right considering the fact that arch doesn't even have a `/usr/share/examples` directory. can you confirm these changes so I can commit them:

```
--- a/srcpkgs/networkmanager-dmenu/template
+++ b/srcpkgs/networkmanager-dmenu/template
@@ -14,6 +14,6 @@ do_install() {
 	vbin networkmanager_dmenu
 	vlicense LICENSE.txt
 	vdoc README.md
-	vdoc config.ini.example
+	vconf config.ini.example
 	vinstall networkmanager_dmenu.desktop 644 /usr/share/applications
 }
```

Reply:
your idea seems right considering the fact that arch doesn't even have a `/usr/share/examples` directory. can you confirm these changes so I can commit them:

```diff
--- a/srcpkgs/networkmanager-dmenu/template
+++ b/srcpkgs/networkmanager-dmenu/template
@@ -14,6 +14,6 @@ do_install() {
 	vbin networkmanager_dmenu
 	vlicense LICENSE.txt
 	vdoc README.md
-	vdoc config.ini.example
+	vconf config.ini.example
 	vinstall networkmanager_dmenu.desktop 644 /usr/share/applications
 }
```

Reply:
correct